networkit/cpp/graph/Graph.cpp

```diff
@@ -223,7 +223,8 @@ void Graph::shrinkToFit() {
 
 void Graph::compactEdges() {
     this->parallelForNodes([&](node u) {
-        if (degreeOut(u) != outEdges[u].size()) {
+        // if (degreeOut(u) != outEdges[u].size()) { //cannot evaluate to true
```

Comment:
Based on an offline discussion: revert the changes and fix compactEdges() in another PR (so leave the function out of tests for now).